### PR TITLE
Creates Beginning Of Application

### PR DESCRIPTION
**Pull request by hhhhhaaaa**
This pull request creates the start of the application.
It contains two pages with text, a general package.json, the base React App setup, Learner's Guild's styling from curriculum, and nodemon.
It was forked off of a base framework by Zubair, and then pushed upstream by me.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "test:watch": "npm run test -- --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LearnersGuild/talent.git"
+  },
+  "keywords": [],
+  "license": "MIT",
+  "dependencies": {
+    "babel-preset-stage-1": "^6.1.18",
```
Comment (jaredatron) [inline package.json]: i think this is a dev dependency

**Review by jaredatron — COMMENTED** [review]
thats a lot of fonts!

**Comment by spencerdezartsmith** [comment]
Hey y’all do we need Redux for this project? From the conversation we had yesterday about the project I would for sure question the need so maybe it’s worth chatting about? Or maybe it doesn’t matter to have it hanging there? Thoughts? Ideas?

**Comment by hhhhhaaaa** [comment]
Howdy!
I'm removing the style folder. If we need the custom fonts, we can procure them at a later time.
I will set babel-present-stage-1 as a dev dependency.
I'm also thinking we do not need Redux, so I'm gonna do my best to remove it from the project, to make things less complicated. This may take some time, but I will see what I can do!

**Review by judytuna — APPROVED** [review]
🏡

**Comment by judytuna** [comment]
Closes https://github.com/LearnersGuild/talent/issues/1

**Pull request merged**